Modules/Project/Services/ProjectService.php

```diff
@@ -307,34 +306,21 @@ public function saveOrUpdateProjectContract($data, $project)
         }
     }
 
-    public function getMailDetailsForProjectManagers()
+    public function getMailDetailsForKeyAccountManager()
```

Comment:
The email is going out for multiple managers some 
```suggestion
    public function getMailDetailsForKeyAccountManagers()
```

Reply:
Also change it every where